.github/workflows/unit_tests.yml

```diff
@@ -1,97 +1,131 @@
 name: unit tests
 
 on:
-  push:     # run on every push or PR to any branch
+  push: # run on every push or PR to any branch
   pull_request:
-  schedule:   # run automatically on main branch each Tuesday at 11am
-  - cron: "0 11 * * 2"
+  schedule: # run automatically on main branch each Tuesday at 11am
+    - cron: "0 11 * * 2"
```

Comment:
not actually 11am, right? (based on this morning's run) should we change either the time or the comment?

Reply:
yes, as we discovered! just to double-check...if it ran at 6am, which is 5hrs earlier than scheduled, then we need to shift the cron forward 5hrs, which would be 4pm or 16, so `0 16 * * 2`. does that sound right?